File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    var result = '';
    for(let i=0; i<my_string.length; i++){
        if(my_string[i]===my_string[i].toUpperCase()){
            result+=my_string[i].toLowerCase()
        }
        else{
            result+=my_string[i].toUpperCase()
        }
    }
    return result
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        StringBuilder result = new StringBuilder();
        
        for(char c:my_string.toCharArray()){
            if(Character.isUpperCase(c)){
                result.append(Character.toLowerCase(c));
            }
            else{
                result.append(Character.toUpperCase(c));
            }
        }
        return result.toString();
    }
}